### PR TITLE
Cd inverted signal

### PR DESCRIPTION
**Pull request by fr0st61te**

### CODE REVIEW

File: arch/arm/boot/dts/aspeed-bmc-opp-vesnin.dts [hunk 1]
```diff
@@ -232,3 +232,16 @@
...
+	status = "okay";
+};
+
+&sdhci1 {
+	status = "okay";
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_sd2_default>;
+	cd-inverted;
+	disable-wp;
```
Comment (AlexanderAmelkin) [inline arch/arm/boot/dts/aspeed-bmc-opp-vesnin.dts:246]: Add a comment "Vesnin BMC uses microSD, there is no WP switch, ignore it". Or at least mention it in the Analysis section in JIRA.

**Review by AlexanderAmelkin — APPROVED** [review]

File: drivers/mmc/host/sdhci-of-aspeed.c [hunk 2]
```diff
@@ -183,6 +199,13 @@ static int aspeed_sdhci_probe(struct platform_device *pdev)
 		goto err_pltfm_free;
 	}
 
+	dev->cd_inverted = 0;
+	host->mmc_host_ops.get_cd = aspeed_get_cd;
```
Comment (nest1ing) [inline drivers/mmc/host/sdhci-of-aspeed.c:203]: I'm a little bit confused by the setting callback here.

While I was trying to find a better place I found that your revision of this repo is mismatched to the OpenBMC's one.
As a result, these changes have a lot of merge conflicts.

**Review by nest1ing — COMMENTED** [review]
So, my question: what is a target of this PR?

File: drivers/mmc/host/sdhci-of-aspeed.c (hunk 2 above)
Comment (AlexanderAmelkin) [reply drivers/mmc/host/sdhci-of-aspeed.c:203]: The target is Linux upstream master.

**Comment by fr0st61te** [comment]
https://lkml.org/lkml/2019/11/20/33

**Pull request closed without merging**